plugin.php

```diff
@@ -2,7 +2,7 @@
 
 /*
   Plugin Name: Gallerya
-  Version: 1.9.4
+  Version: 2.0.9
```

Comment:
Why this massive version plugin version change? Updating the dependencies in minor versions should not update the plugin in a major version. Please see [semver.org](http://semver.org) for how to versioning.